server/src/controllers/projectController.js

```diff
@@ -78,3 +100,22 @@ module.exports.getOneProject = (req, res) => {
       res.status(400).json(err);
     });
 };
+
+/**
+ * Create One Project
+ * @param req The HTTP Request Object
+ * @param res The HTTP Response Object
+ */
+module.exports.createOneProject = (req, res) => {
```

Comment:
It's good that you're thinking about creating the project itself, but I'm not sure that it is necessary for this module. Since it's focus is rewards, I think you could just include a "foreign key" in your project and fake it.

Reply:
@quinnfinite I think this kind of answers my question from earlier today. I want to simplify this as much as I can since there is enough complexity as it is. Thanks for this suggestion.